frontend/src/workspace/buildingTools/ValNode.js

```diff
@@ -96,10 +96,10 @@ function ValNode(props) {
     return (
       <Image
         image={image}
-        x={nodeDimensions.functionTrashX - valueWidth * 2/3}
+        x={nodeDimensions.functionTrashX-50}
```

Comment:
why are these numbers instead of ratios? applies throughout this file. ratios seem like a safer overall choice.

Reply:
I think this is a result from when we didn't realize that git add * was a bad idea, and was meant to just be for my own formatting. Feel free to discard.